README/ReleaseNotes/v630/index.md

```diff
@@ -369,6 +369,15 @@ the behavior consistent with `RooAbsPdf::fitTo()`. Same applies to
   Have to be used if called from other web-based widget to avoid logical dead-locks.
   In case of normal canvas just canvas->Update() is performed.
 
+## Machine Learning integration
+
```

Comment:
We can probably do without a bulleted list here, just a regular paragraph?

Reply:
Yes in principle but since this is a backport I would keep it as it is for this time